src_web/kamaho-shokusu/src/Command/ExpireTrialTenantsCommand.php

```
@@ -0,0 +1,76 @@
+use Cake\Console\ConsoleOptionParser;
+use Cake\I18n\DateTime;
+use Cake\ORM\TableRegistry;
+
+/**
+ * 日次バッチ: トライアル期限切れテナントを suspended に移行する。
+ *
+ * 実行例:
+ *   bin/cake expire_trial_tenants
+ *   bin/cake expire_trial_tenants --dry-run
+ */
+class ExpireTrialTenantsCommand extends Command
```

PHPStan (2.2.5)

[error] Class App\Command\ExpireTrialTenantsCommand extends unknown class Cake\Command\Command.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)